src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -1,53 +1,84 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.kauailabs.navx.frc.AHRS;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
 
+import edu.wpi.first.math.estimator.DifferentialDrivePoseEstimator;
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.simulation.DifferentialDrivetrainSim;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+import frc.robot.constants.Constants;
+import frc.robot.constants.DriveConstants;
 
 public class Drivetrain extends SubsystemBase {
   
-  private CANSparkMax leftMotor1;
-  private CANSparkMax leftMotor2;
-  private CANSparkMax rightMotor1;
-  private CANSparkMax rightMotor2;
+  // private TalonFX leftMotor1;
+  // private TalonFX leftMotor2;
+  // private TalonFX rightMotor1;
+  // private TalonFX rightMotor2;
+  CANSparkMax leftMotor1;
+  CANSparkMax rightMotor1;
+
+  DifferentialDrivetrainSim cakeOrFake;
```

Comment:
Instance variables should be private. Also, name this something more descriptive.

Reply:
You changed it to be more descriptive, but make all of them private.